deploy/ocs-operator/manifests/ux-backend-reviews.yaml

```diff
@@ -0,0 +1,53 @@
+---
```

Comment:
why do we need two files and also this much verbosity, we can club all of them in a single manifest w/ a ClusterRole and ClusterRoleBinding